_posts/convars/2021-12-10-mom_hud_speedometer_fps.md

```diff
@@ -0,0 +1,15 @@
+---
+title: mom_hud_speedometer_fps
+category: var
+tags:
+  - hud
+  - speedometer
+minimum_value: 1
+maximum_value: 0 (Uncapped)
```

Comment:
The cvar doesn't have any bounds (could even be negative even, though that would effectively be the same as 0) so I'd omit both of these.

Reply:
Resolved in [`4aa6982` (#232)](https://github.com/momentum-mod/docs/pull/232/commits/4aa6982a1d51a860b384bffe7b3e84576d6f4df7)